applications/feedback_sentiment_analyzer/cdk/lib/env.ts

```diff
@@ -0,0 +1,23 @@
+let APP_NAME: string, APP_EMAIL: string, APP_LANG: string;
+
+if (!process.env.APP_NAME) {
+  throw new Error("APP_NAME is not defined");
```

Comment:
These are end-user cases and really annoying when they happen in PAM, so let's not throw here (we know where the problem is) and instead

```
console.error("FSA_NAME is not defined");
process.exit(1);
```